components/common/BoardEditor/focalboard/src/mutator.ts

```
@@ -651,6 +651,19 @@ export class Mutator {
     return this.updateBlocks(newBlocks, oldBlocks, description);
   }
 
+  async updateProperty(board: Board, propertyId: string, updatedProperty: IPropertyTemplate) {
+    const newBoard = createBoard({ block: board });
+    const cardPropertyIndex = board.fields.cardProperties.findIndex((o: IPropertyTemplate) => o.id === propertyId);
+
+    if (cardPropertyIndex === -1) {
+      return;
```

should throw an error